anypoint-signin-aware.js

```diff
@@ -44,16 +53,26 @@ export const AnypointAuth = {
       AnypointAuth._redirectUri = val;
     }
   },
-  // OAuth2 authorization type
-  authType: 'implicit',
+  // OAuth2 authorization type. e.g. implicit, authorization_code, etc.
+  // By default, the authorization type is authorization_code.
+  _authType: GRANT_TYPES.AUTH_CODE,
+  get authType() {
+    return AnypointAuth._authType;
+  },
+  set authType(val) {
+    if (val && val !== AnypointAuth._authType) {
+      AnypointAuth._authType = val;
```

Comment:
Why repeat AnypointAuth._authType = val;
twice? You always do it.

Reply:
Ah, good catch, was copy and paste, let me fix it for other places too